tests/fxl-properties-001/OPS/package.opf

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout Properties</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-09-22</dc:date>
+    <dc:description>
+    When the rendition:layout property is set to pre-paginated, Reading Systems MUST NOT include space between the adjacent content slots when rendering Synthetic Spreads.
+    </dc:description>
+    <dc:identifier id="pub-id">https://w3c.github.io/epub-specs/epub33/rs/#layout</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:layout pre-paginated Reading System Rendering</dc:title>
```

Comment:
Like description, I think this should describe the *state* of the test, not what it tests. See example titles at https://iherman.github.io/epub-testing/results/tests.html#sec-test-tables and please match.